bin/node/cli/src/browser.rs

```diff
@@ -19,26 +19,26 @@ use log::info;
 use wasm_bindgen::prelude::*;
 use sc_service::Configuration;
 use browser_utils::{
-	Transport, Client,
+	Client,
 	browser_configuration, set_console_error_panic_hook, init_console_log,
 };
 use std::str::FromStr;
 
 /// Starts the client.
 #[wasm_bindgen]
-pub async fn start_client(chain_spec: String, log_level: String, wasm_ext: Transport) -> Result<Client, JsValue> {
-	start_inner(chain_spec, log_level, wasm_ext)
+pub async fn start_client(chain_spec: String, log_level: String) -> Result<Client, JsValue> {
+	start_inner(chain_spec, log_level)
 		.await
 		.map_err(|err| JsValue::from_str(&err.to_string()))
 }
 
-async fn start_inner(chain_spec: String, log_level: String, wasm_ext: Transport) -> Result<Client, Box<dyn std::error::Error>> {
+async fn start_inner(chain_spec: String, log_level: String) -> Result<Client, Box<dyn std::error::Error>> {
 	set_console_error_panic_hook();
 	init_console_log(log::Level::from_str(&log_level)?)?;
 	let chain_spec = ChainSpec::from_json_bytes(chain_spec.as_bytes().to_vec())
 		.map_err(|e| format!("{:?}", e))?;
 
-	let config: Configuration<_, _> = browser_configuration(wasm_ext, chain_spec)
+	let config: Configuration<_, _> = browser_configuration(chain_spec)
```

Comment:
is `: Configuration<_, _>` annotation still required ?

if you tell me how to compile it locally, I'll check myself :)

Reply:
`cargo check --target wasm32-unknown-unknown -p node-cli --no-default-features --features browser` should do it.

Reply:
Done